pb/rpc.proto

```diff
@@ -11,45 +11,44 @@ message RPC {
 		optional string topicid = 2;
 	}
 
-	optional ControlMessage control = 3;
+    optional ControlMessage control = 3;
 }
 
 message Message {
 	optional bytes from = 1;
 	optional bytes data = 2;
 	optional bytes seqno = 3;
 	repeated string topicIDs = 4;
-	optional bytes signature = 5;
```

Comment:
Were these two lines meant to be deleted?

Reply:
Oh, no, looks like that line has been added some time recently, I was editing from an older file.